website/content/90-cleanups/_index.md

```diff
@@ -6,5 +6,5 @@ chapter = true
 <center><h2>Cleaning Up</h2></center>
 
 <div style="text-align: justify">
-When you have finished with this workshop, remember to clean up all those AWS resources that you created using AWS CloudFormation. Use the AWS CloudFormation console or AWS CLI to delete the stack named <b>Lake-Formation-WF</b> and <b>Lake-Formation-With-EMR-WF</b>.
+When you have finished with this workshop, remember to clean up all those AWS resources that you created using AWS CloudFormation. You have to delete the content in the S3 buckets created part of this exercise before deleting the CloudFormation Stacks. Use the AWS CloudFormation console or AWS CLI to delete the stack named <b>Lake-Formation-WF</b> and <b>Lake-Formation-With-EMR-WF</b>.
```

Comment:
Provide a link on emptying the folder and then deleting the S3 bucket

Reply:
Provided more instructions and added screen to support this activity.

Reply:
CFN delete will remove the resources (the first sentence is not right). The problem is only with S3 content (and VPCs sometimes when they have ENIs attached) as CFN wont remove the bucket.